nodejs/week3/meal_sharing_app/src/backend/api/reviews.js

```diff
@@ -0,0 +1,61 @@
+const express = require("express");
+const app = express();
+const router = express.Router();
+const knex = require("../database");
+
+app.use(express.json());
+app.use(express.urlencoded({
+  extended: true
+}));
+
+router.get("/", async function (req, res) {
+  const totalReviews = await knex.select("*").table("review");
+  res.send(totalReviews);
+});
+
+// http://localhost:3000/api/reviews
+
+router.post("/", async function (req, res) {
+  const newReview = {
+    "id": req.params.id,
+    "title": req.query.title,
+    "description": req.query.description,
+    "meal_id": req.query.meal_id,
+    "stars": req.query.stars,
+    "created_date": req.query.created_date
+  }
+  await knex("review").insert(newReview);
+  res.send("The given review has been added");
+});
+
+//http://localhost:3000/api/reviews?title=russian&description=super&meal_id=6&stars=5&created_date=2020-06-27
+
+router.get("/:id", async function (req, res) {
+  const thisReviewId = await knex("review").where({
```

Comment:
You are doing a great job with the variables! Just to say. Remember to give variable names after what the variable represents. This simply represents a review. Adding `this` to the varible name does not add any information. Adding id is a little dangerous, because another developer would assume the variable is a number (an id). But it is actually an object representing a review. Hope that makes sense